code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -585,15 +585,6 @@
 	construction_time = 100
 	category = list("Exosuit Equipment")
 
-/datum/design/mech_cable_layer
-	name = "Exosuit Engineering Equipment (Cable Layer)"
-	id = "mech_cable_layer"
-	build_type = MECHFAB
-	build_path = /obj/item/mecha_parts/mecha_equipment/cable_layer
-	materials = list(/datum/material/iron=10000)
-	construction_time = 100
-	category = list("Exosuit Equipment")
-
```

Comment:
?

Reply:
I tried keeping it the best i could, seemingly all the code that this requires to execute was wiped?